spec/rubocop/cop/lint/void_spec.rb

```diff
@@ -65,6 +65,38 @@
     RUBY
   end
 
+  it 'registers an offense for void literal in a method definition' do
+    expect_offense(<<-RUBY.strip_indent)
+      def something
+        42
+        ^^ Literal `42` used in void context.
+        42
+      end
+    RUBY
+  end
+
+  it 'registers two offenses for void literals in an initialize method' do
+    expect_offense(<<-END.strip_indent)
```

Comment:
Use `RUBY` everywhere consistently. We really need a cop to warn for usages of `END`.